libs/sdk-ui-pivot/src/types.ts

```diff
@@ -31,7 +31,7 @@ export interface IMenu {
 /**
  * @public
  */
-export type DefaultColumnWidth = "viewport" | "unset"; // | "auto"  | number; can be added later see ONE-4276
+export type DefaultColumnWidth = "viewport" | "unset" | "autoresizeAll"; // | "auto"  | number; can be added later see ONE-4276
```

Comment:
does the comment still make sense?

Reply:
I suppose that it can be removed. It is some leftover from the original autoresize and it does not make sense now since we have manual resizing.

Reply:
Removed.